src/multivariate/composite.jl

```diff
@@ -0,0 +1,203 @@
+# Composite Multivariate distribution
+
+## Generic Composite multivariate distribution class
+
+module CompositeDistributions
```

Comment:
Putting this into a separate module really complicates the code (e.g. the many lines of imports shown below). I would not do this in production code.